frontend_app_agenda/package.json

```diff
@@ -4,12 +4,16 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "servdev": "NODE_ENV=development webpack-dev-server --watch --colors --inline --hot --progress",
-    "servdevwindoz": "set NODE_ENV=development&& webpack-dev-server --watch --colors --inline --hot --progress",
+    "servdev": "NODE_ENV=development webpack-dev-server --config webpack.tracim.config.js --watch --colors --inline --hot --progress",
+    "servdevwindoz": "NODE_ENV=development webpack-dev-server --config webpack.tracim.config.js --watch --colors --inline --hot --progress",
```

Comment:
I don't think this command will work on windows.
The `set` and `&&` have its uses

Reply:
it does not work on Yarn. Yarn does not use bash and `set` does not on Yarn's interpreter. This is why I changed this.

Now, either we drop support for building on Windows, or we need to test the build on it. It's already broken without this PR.

Reply:
supposedly, the new line will have more chance to work on windows because Yarn embeds its own shell interpreter explicitly for windows support

Reply:
See https://dev.to/arcanis/introducing-yarn-2-4eh1#normalized-shell @Skylsmoi

Reply:
oh, that's good to know.
Then, I agree that your version might be better.
Though, it might be wise to test it on windows before merging this.
Or just drop windows support

Reply:
I'd say let's drop the Mic(rosoft) until we test it.
I can't test on windows (I don't have a Windows machine), and the build scripts for Windows and regular OSes are now (almost?) identical anyway with Yarn.

Reply:
@raphj @Skylsmoi we have windows VM at algoo. I think it's not your role to decide what developement environment is supported: it may have an impact on the business. I do not say "you do not have your word on it" but I'd like you not to decide without to talk to others (especially me).

Tracim is not a personal side project. If you make choices that has impact on the business, people doing business at algoo must be in the decision loop (and probably be the ones who decide).

Not: this is the same thing for which linux distribution is supported, which database is supported, which python version is supported, etc.

Reply:
Thanks for the reminder @lebouquetin and sorry for that. I worded things as we were dropping the support for Windows and I should have not, thanks for calling me out on this.

Now, the decision being discussed here is really about dropping the specific flags that were needed to support WIndows with npm and are not needed anymore with Yarn, and are currently broken in develop (support for Windows is currently not mentioned anywhere and I had no clue we did before yesterday - I did not understand what `-w` and `windoz` were, I tried to keep them working the way they were when switching to Yarn and actually failed).

With Yarn there is no need to use a `-w` flag so a different shell syntax is used on Windows in NPM scripts anymore since the same "shell" interpreter is used across platforms with Yarn.

 (and for sure, this information needs to be put into the ticket and I will do this when this discussion is settled)

Now, we need to decide whether we should (support / keep supporting) building Tracim on Windows and if so, test and document the process, independently of this PR. Should I open a ticket? (I mentioned WSL to @Skylsmoi as an idea to support Windows painlessly).